atuin-client/src/encryption.rs

```diff
@@ -81,22 +81,22 @@ pub fn encode_key(key: &Key) -> Result<String> {
     Ok(buf)
 }
 
-pub fn decode_key(key: String) -> Result<Key> {
+#[allow(clippy::similar_names)]
```

Comment:
clippy suggests to rename `mbuf` to `buf` which might actually be a false suggested solution as this obviously has to fail. Any suggestions for `mbuf` instead?